### PR TITLE
ENH: Audit EXISTS Docs

### PR DESCRIPTION
**Pull request by vanshavenger**
Screenshot 2024-10-21 at 10 46 33 PM
Screenshot 2024-10-21 at 10 46 21 PM
Screenshot 2024-10-21 at 10 46 27 PM
Screenshot 2024-10-21 at 10 46 15 PM
#826 

CC @lucifercr07 
Let me know if i missed anything. Happy to contribute.

### CODE REVIEW

**Review by apoorvyadav1111 — CHANGES_REQUESTED** [review]
Thanks for the changes, left some minor comments.

File: docs/src/content/docs/commands/EXISTS.md [hunk 1]
```diff
@@ -1,89 +1,89 @@
...
-- If one or more of the specified keys exist, the command returns the count of those keys.
+| Condition                                      | Return Value                                      |
+|------------------------------------------------|---------------------------------------------------|
+| None of the specified keys exist               | `0`                                               |
+| One or more specified keys exist               | Integer representing the count of keys that exist |
 
 ## Behaviour
+- The `EXISTS` command checks whether the specified keys are present in the database.
+- If one key is specified, the command returns `1` if the key exists and `0` if it does not.
+- If multiple keys are specified, the command returns the count of keys that exist among the given keys.
```
[the remark this answers is not in the document]
Comment (vanshavenger) [reply docs/src/content/docs/commands/EXISTS.md:29]: The command returns 1 if a single specified key exists and 0 if it does not, or the count of valid keys among multiple specified keys.

Works?

File: docs/src/content/docs/commands/EXISTS.md [hunk 2]
```diff
@@ -1,89 +1,89 @@
...
+   - Occurs when no key is provided.
 
-1. `Wrong Type of Arguments`: If the command is not provided with at least one key, DiceDB will return a syntax error.
-
-   - `Error Message`: `(error) ERR wrong number of arguments for 'exists' command`
-
-2. `Non-String Keys`: If the keys provided are not strings, DiceDB will raise a type error.
-
-   - `Error Message`: `(error) ERR value is not a valid string`
+2. **Invalid key type**:
```
[the remark this answers is not in the document]
Comment (vanshavenger) [reply docs/src/content/docs/commands/EXISTS.md:37]: Fixing it and Pushing new commits

**Comment by vanshavenger** [comment]
@apoorvyadav1111 can you check now once

File: docs/src/content/docs/commands/EXISTS.md [hunk 3]
```diff
@@ -25,18 +25,17 @@ EXISTS key [key ...]
...
 
 ## Errors
 1. **Wrong number of arguments**:
    - Error Message: `(error) ERR wrong number of arguments for 'exists' command`
    - Occurs when no key is provided.
 
-2. **Invalid key type**:
-   - Error Message: `(error) ERR value is not a valid string`
-   - Occurs when a non-string value is provided as a key.
+2. **Wrong type of value or key**:
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/EXISTS.md:36]:
```suggestion
2. `Wrong type of value or key`:
```

File: docs/src/content/docs/commands/EXISTS.md [hunk 4]
```diff
@@ -25,18 +25,17 @@ EXISTS key [key ...]
 
 ## Behaviour
 - The `EXISTS` command checks whether the specified keys are present in the database.
-- If one key is specified, the command returns `1` if the key exists and `0` if it does not.
-- If multiple keys are specified, the command returns the count of keys that exist among the given keys.
+- Returns 1 or 0, or for multiple keys returns the count of existing keys.
 - The command performs a read-only operation and does not modify the database.
 
 ## Errors
 1. **Wrong number of arguments**:
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/EXISTS.md:32]:
```suggestion
1. `Wrong number of arguments`:
```

**Review by apoorvyadav1111 — CHANGES_REQUESTED** [review]
left two minor comments. can be considered a nit but this makes it consistent with every other doc.

**Review by apoorvyadav1111 — APPROVED** [review]

**Pull request merged**